.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Our contact information
+    url: https://thalia.nu/association/committees/42/
+    about: For any questions not related to this repository
+  - name: Security issues
+    url: https://thalia.nu/contact
```

Comment:
I don't know if https://thalia.nu/contact is the best contact information for security issues.

Reply:
I have completely removed it